homeassistant/components/androidtv_remote/const.py

```diff
@@ -4,3 +4,111 @@
+    "com.jio.media.stb.ondemand": "Jio Cinema",
+    "org.xbmc.kodi": "Kodi",
+    "com.google.android.tv": "Live Channels",
+    "de.telekom.magentatv.atv": "Magenta TV",
+    "org.samsonsen.nederlandse.radio.holland.nl": "Mijn Radio",
+    "tv.molotov.app": "Molotov",
+    "tv.mrmc.mrmc": "MrMC",
+    "tv.mrmc.mrmc.lite": "MrMC Lite",
+    "com.mxtech.videoplayer.ad": "MX Player",
+    "com.netflix.ninja": "Netflix",
+    "nl.nlziet": "NLZIET",
+    "nl.nos.app": "NOS",
+    "nl.uitzendinggemist": "NPO",
+    "com.orange.ocsgo": "OCS",
+    "com.google.android.play.games": "Play Games",
+    "com.google.android.music": "Play Music",
+    "com.android.vending": "Play Store",
+    "com.google.android.videos": "Play Movies & TV",
+    "com.plexapp.android": "Plex",
+    "com.amazon.amazonvideo.livingroom": "Prime Video",
+    "com.amazon.firebat": "Prime Video (FireTV)",
+    "com.android.tv.settings": "Settings",
+    "com.teamsmart.videomanager.tv": "SmartTube",
+    "com.liskovsoft.videomanager": "Smart YouTube TV",
+    "com.sony.dtv.scrums.action": "Action Menu",
+    "com.sony.dtv.osat.album": "Album",
+    "com.sony.dtv.braviasyncmenu": "Sync Menu",
+    "com.sony.dtv.bravialifehack": "BRAVIA Tutorials",
+    "com.sony.dtv.discovery": "Discover",
+    "com.sony.dtv.smarthelp": "Help",
+    "com.vewd.core.integration.dia": "Internet Browser",
+    "com.sonyliv": "SonyLIV",
+    "com.sony.dtv.osat.music": "Music",
+    "com.sony.dtv.networkapp.wifidirect": "Screen mirroring",
+    "com.sony.dtv.sonyselect": "Sony Select",
+    "com.sony.dtv.timers": "Timers",
+    "com.sony.dtv.tvx": "TV",
+    "com.sony.dtv.osat.video": "Video",
+    "de.sport1.firetv.video": "Sport 1",
+    "com.spotify.tv.android": "Spotify",
+    "com.valvesoftware.steamlink": "Steam Link",
+    "com.amazon.webapps.nbc.syfy": "Syfy",
+    "tv.perception.clients.tv.android": "T-2 TV",
+    "com.ted.android.tv": "TED",
+    "ar.tvplayer.tv": "TiviMate",
+    "tunein.player": "TuneIn Radio",
+    "de.cyberdream.dreamepg.tvh.tv.player": "DreamPlayer TVHeadend",
+    "tv.twitch.android.app": "Twitch",
+    "tv.twitch.android.viewer": "Twitch (FireTV)",
+    "com.vevo.tv": "Vevo",
+    "com.mtvn.vh1android": "VH1",
+    "com.vimeo.android.videoapp": "Vimeo",
+    "net.gtvbox.videoplayer": "Vimu Media Player",
+    "net.gtvbox.vimuhd": "Vimu Media Player",
+    "org.videolan.vlc": "VLC",
+    "com.phonegap.voyo": "VOYO",
+    "com.ellation.vrv": "VRV",
+    "de.exaring.waipu.firetv.live": "Waipu TV",
+    "com.turner.tnt.android.networkapp": "Watch TNT",
+    "com.google.android.youtube.tv": "YouTube",
+    "com.amazon.firetv.youtube": "YouTube (FireTV)",
+    "com.google.android.youtube.tvkids": "YouTube Kids",
+    "com.amazon.firetv.youtube.kids": "YouTube Kids (FireTV)",
+    "com.google.android.youtube.tvmusic": "YouTube Music",
+    "com.google.android.youtube.tvunplugged": "YouTube TV",
+    "com.graymatrix.did": "ZEE5",
+    "com.ziggo.tv": "Ziggo GO TV",
+}
```

Comment:
Grabbed from https://github.com/JeffLIrion/python-androidtv/blob/3b9f7cbabab0049d21c9ddf6c5bcc49d7c415789/androidtv/constants.py#L391-L494

I think in the follow up PRs we can add the ability to customize this list for users.

Reply:
Yes, I agree with you - i removed the list. But what do you think about the ability for users to create their own list in `OptionsFlow`, as it was done in the old Android TV integration? We can do it in the future. Or is it useless?